src/validation/custom.validation.ts

```diff
@@ -452,22 +454,55 @@ export const checkAtLeastOneFieldHasValue = (errMsg: string, ...fields: any[]) =
   throw new Error(errMsg);
 };
 
-export const checkTrustFields = (trustsJson: string) => {
+const isSubmitTrustSelected = (session: Session, action: string = ""): boolean => {
+  const appData: ApplicationData = getApplicationData(session);
+  return action === "submit" && (appData.trusts || [] ).length > 0;
+};
+
+export const checkTrustBO = (req) => {
```

Comment:
I think method name should be `checkTrustBOs`. It's called from the validator and is linked to the "beneficialOwners" field.